tests/e2e/opampbridge/00-assert.yaml

```diff
@@ -0,0 +1,48 @@
+apiVersion: apps/v1
+kind: Deployment
+metadata:
+  name: test-opamp-bridge
+spec:
+  template:
+    spec:
+      containers:
```

Comment:
should there be an image in here that we check for?

Reply:
I don't think so. In other test cases also it is not there.